CHANGELOG.md

```diff
@@ -28,7 +28,10 @@
 * [ENHANCEMENT] Compactor: added the configurable limit `compactor.block-upload-max-block-size-bytes` or `compactor_block_upload_max_block_size_bytes` to limit the byte size of uploaded or validated blocks. #4680
 * [ENHANCEMENT] Querier: reduce CPU utilisation when shuffle sharding is enabled with large shard sizes. #4851
 * [ENHANCEMENT] Packaging: facilitate configuration management by instructing systemd to start mimir with a configuration file. #4810
-* [ENHANCEMENT] Store-gateway: reduce memory allocations when looking up postings from cache. #4861 #4869
+* [ENHANCEMENT] Store-gateway: reduce memory allocations when looking up postings from cache. #4861 #4869 #4962 #5047
+* [ENHANCEMENT] Store-gateway: retain only necessary bytes when reading series from the bucket. #4926
+* [ENHANCEMENT] Ingester: improve handling of the shutdown marker. #4985
```

Comment:
Yes, this is very generic, but I'm not sure if your suggestion captures the gist of the change well. Maybe @56quarters has a good suggestion?

Reply:
I've read the PR and it also affects the store gateway. So I would say something like
```suggestion
* [ENHANCEMENT] Ingester, store-gateway: clear the shutdown marker after a successful shutdown to enable reusing their persistent volumes in case the ingester or store-gateway is restarted. #4985
```